selfdrive/test/process_replay/test_regen.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import unittest
+
+from parameterized import parameterized
+
+from openpilot.selfdrive.test.process_replay.regen import regen_segment
+from openpilot.selfdrive.test.process_replay.process_replay import check_openpilot_enabled, CONFIGS
+from openpilot.selfdrive.test.openpilotci import get_url
+from openpilot.tools.lib.logreader import LogReader
+from openpilot.tools.lib.framereader import FrameReader
+
+EXCLUDED_PROCESSES = {"dmonitoringd", "dmonitoringmodeld"}
```

Comment:
why is DM disabled? does the extra framereader significantly slow things down?

Reply:
The segments used are public and are missing dcamera.hevc